insights/client/cert_auth.py

```diff
@@ -23,7 +23,7 @@ class rhsmCertificate:
     try:
         PATH = RHSM_CONFIG.get('rhsm', 'consumerCertDir')
     except:
-        pass
+        PATH = "/etc/pki/consumer/"
```

Comment:
Maybe I would add here some `logger.warning()` that it failed to get `consumerCertDir` from `RHSM_CONFIG`

Non-blocking

Reply:
That would only happen if you didn't have subscription-manager installed, which isn't a supported configuration. Other parts of the code already raise errors about it:

https://github.com/RedHatInsights/insights-core/blob/53b5ff55ce3ef6b47b444089f47c7357694a566a/insights/client/connection.py#L151-L157

Reply:
Okay, then just please rebase